talk/owt/sdk/base/customizedvideodecoderproxy.cc

```diff
@@ -25,25 +25,25 @@ bool CustomizedVideoDecoderProxy::Configure(const Settings& codec_settings) {
   if (external_decoder_) {
     if (codec_type_ == kVideoCodecH264 &&
         external_decoder_->InitDecodeContext(VideoCodec::kH264)) {
-      return WEBRTC_VIDEO_CODEC_OK;
+      return true;
```

Comment:
In old version, InitDecoder returns int32_t, but in new version, a new function Configure replaces InitDecoder, which returns bool. So it's okay to return true without any warnings. Since WEBRTC_VIDEO_CODEC_OK is defined as 0, it returned false and caused the issue you reported.